.gitignore

```diff
@@ -38,6 +38,7 @@ config/overlays/development/configmap/inferenceservice_patch.yaml
 config/overlays/dev-image-config/inferenceservice_patch.yaml
 config/overlays/local/
 .ko.yaml
+docs/samples/logger/local/iris**
```

Comment:
I'd prefer to not codify this :P. Just don't check it in.

Reply:
:+1: have changed it